### ISSUE TITLE
Used automationName:Flutter, but There's no accessibility id or resource id on Android System

### ISSUE THREAD

**Issue by Saki2012** [appium/appium-inspector#651]
## The problem: Can't find accessibility id or resource id on android

I have an app in flutter(dart) like this
(used enableFlutterDriverExtension() ,and output debug apk, too)
![image](https://user-images.githubusercontent.com/32598251/200776963-e6a33971-948f-443c-beca-580711aaaae6.png)
when I inspect on Android System,the element don't have any id but text, why?
<img width="1280" alt="image" src="https://user-images.githubusercontent.com/32598251/200777606-0554da0c-4488-4cd0-8510-f44c9d13d61d.png">

for the iOS,there's work.
<img width="1280" alt="image" src="https://user-images.githubusercontent.com/32598251/200779302-01da388c-f85f-4fa0-be42-7a7cf23c50c4.png">

Should I Used other tools to find elements and test?

## Environment

- I am running Appium Desktop version _<1.22.3>_.
- I am on:
    - [x] Windows
    - [x] Android
    - [x] Mac
    - [ ] Linux

- Android's capabilities
![image](https://user-images.githubusercontent.com/32598251/200779626-81a61f82-ac5a-4968-b1b3-def52f191e64.png)
- iOS's capabilities
![image](https://user-images.githubusercontent.com/32598251/200779722-47e7fb41-c6c8-4999-85d5-e190d03b79c8.png)

**Comment by brett-james-rocketlab** [comment]
That's not Appium Inspectors fault, It's Android's for not exposing that properly.

I've done a deep dive on this for React Native here that I will paste, which helps give some insight:

> `This guide is thorough and goes in detail. https://dev.to/nextlevelbeard/an-end-to-the-abuse-of-accessibility-ids-5d2j
The short version is if you were to hardcode it for android it would be:
<textinput
... other params here....
testID='com.your.application:id/username'
nativeID='username'
/>
I also put in nativeID because apparently it should also do the same - HOWEVER, it did not work when I was trying it - but testID did!
The example website also has a helper function that you can use, so depending on the platform (iOS doesn't want the bundle id before hand), it returns the correct thing as well too.`
Another link: 
https://dev.to/nextlevelbeard/an-end-to-the-abuse-of-accessibility-ids-5d2j

In short, what you want to do is to set either the testID, or ideally the nativeID. 

Now the trouble here with Flutter is that it does not use the native widgets to render something, so this is where I am lost and would be keen to see what your research shows too. They need to in short make use of the native operating systems ability to hook into those properties to have them be identifiable in the normal ways, allowing for easy testing.

**Comment by jlipps** [comment]
thanks for the input @brett-james-rocketlab. this is correct. please add any more details here if you find any w.r.t. how flutter driver sets these labels, @Saki2012
